app/static/src/app/components/sensitivity/SensitivityTab.vue

```diff
@@ -11,8 +11,15 @@
             >
         </div>
         <action-required-message :message="updateMsg"></action-required-message>
-        <sensitivity-traces-plot v-if="tracesPlot" :fade-plot="!!updateMsg"></sensitivity-traces-plot>
-        <sensitivity-summary-plot v-else :fade-plot="!!updateMsg"></sensitivity-summary-plot>
+        <template v-for="(config, index) in graphConfigs" :key="config.id">
+            <sensitivity-traces-plot
+                v-if="tracesPlot"
```

Comment:
Yeah it can, though I think the summary plot is fairly rarely used compared to the traces plot, so I'm inclined to leave that to render as required for now.